src/app/components/Document.jsx

```diff
@@ -1,7 +1,18 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import { AfterRoot, AfterData } from '@jaredpalmer/after';
 import { ServerStyleSheet } from 'styled-components';
 
+const resourceHints = () => (
+  <Fragment>
+    <link rel="preconnect" href="//ichef.bbci.co.uk" />
```

Comment:
@sareh why don't these include 'https' we should be https only? Also don't we need crossorigin?

Reply:
I've addressed this in a new commit, and added tests.